changelog/8.8.0.yaml

```diff
@@ -0,0 +1,320 @@
+      pr:
+        - "2456"
+      issue:
+        - "2450"
+      timestamp: 1680764383
+      file:
+        name: 1680764383-Respecting-logging.level-settings.yaml
+        checksum: b4c689aa41884020f23848adab7b586d886c3659
+    - kind: bug-fix
+      summary: An empty proxy from the policy don't override the proxy set by --proxy-url.
+      description: ""
+      component: action handlers
+      pr:
+        - https://github.com/elastic/elastic-agent/pull/2468
+      issue: []
+      timestamp: 1681211832
+      file:
+        name: 1681211832-empty-proxy-from-the-policy-don't-overrides-the-proxy-set-by---proxy-url.yaml
+        checksum: b5edcf2189cdf6d4a3666f179333fc6398fedfcb
+    - kind: bug-fix
+      summary: Ensure local bin directory exists on Mac OS during installation.
+      description: |
+        During installation of the Elatic Agent on Mac OS ensure that the /usr/local/bin path exists before
+        creating the /usr/local/bin/elastic-agent symlink.
+      component: ""
+      pr:
+        - https://github.com/elastic/elastic-agent/2490
+      issue: []
+      timestamp: 1681311024
+      file:
+        name: 1681311024-Ensure-local-bin-directory-exists-on-Mac-OS-during-installation..yaml
+        checksum: 648894f466ad911ee06be9c3b4b2477fad3488bc
+    - kind: enhancement
+      summary: Make the language used in CLI commands more consistent.
+      description: ""
+      component: agent
+      pr:
+        - https://github.com/elastic/elastic-agent/pull/2496
+      issue: []
+      timestamp: 1681340196
+      file:
+        name: 1681340196-More-consistent-language-in-CLI-commands.yaml
+        checksum: e91dd7f091ca01562479ca131ddfa5d9503e76c9
+    - kind: bug-fix
+      summary: Report not bootstrapped with fleet-server as an error.
+      description: ""
+      component: ""
+      pr:
+        - https://github.com/elastic/elastic-agent/pull/2505
+      issue:
+        - https://github.com/elastic/elastic-agent/issues/2170
+      timestamp: 1681742558
+      file:
+        name: 1681742558-Report-not-bootstrapped-with-fleet-server-as-an-error.yaml
+        checksum: 2c601eb490be451cc5e9e022b8d82d6bc225016e
+    - kind: bug-fix
+      summary: Re-add lumberjack input to the Filebeat spec.
+      description: ""
+      component: spec
+      pr:
+        - https://github.com/elastic/elastic-agent/pull/2511
+      issue: []
+      timestamp: 1681842199
+      file:
+        name: 1681842199-re-add-lumberjack-input-spec.yaml
+        checksum: a0bb03fb0a7739946d631eece1dc0c5a807038e6
+    - kind: feature
+      summary: Allow users to specify custom base paths during installation
+      description: ""
+      component: agent
+      pr:
+        - https://github.com/elastic/elastic-agent/pull/2500
+      issue: []
+      timestamp: 1681940139
+      file:
+        name: 1681940139-Allow-users-to-specify-custom-base-paths-during-installation.yaml
+        checksum: 7f2cf8493a3d6cf61ff9f8117b965ff2ba327046
```

Comment:
It won't be officially released in 8.8.0 so we should remove it from the changelog.